index.html

```diff
@@ -4,10 +4,35 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
   <script defer src="./js/app.js"></script>
+  <link href="./css/cardstarter.css" rel="stylesheet">
   <link rel="stylesheet" href="./css/style.css">
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
```

Comment:
Take away the `/` from the start of the path on this and your favicon will work!